### PR TITLE
Add Ceph kamelets

### PR DESCRIPTION
**Pull request by claudio4j**
https://github.com/apache/camel-kamelets/issues/1003

### CODE REVIEW

**Comment by claudio4j** [comment]
@christophd for review.

**Review by davsclaus — APPROVED** [review]

File: kamelets/ceph-sink.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    camel.apache.org/kamelet.group: "Ceph"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Ceph Sink"
+    description: |-
+      Upload data to an Ceph Bucket managed by a Object Storage Gateway.
+
+      In the header, you can optionally set the `file` / `ce-partition` property to specify the name of the file to upload.
```
Comment (oscerd) [inline kamelets/ceph-sink.kamelet.yaml:36]: ce-file and not ce-partition

File: library/camel-kamelets/src/main/resources/kamelets/ceph-sink.kamelet.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    camel.apache.org/kamelet.group: "Ceph"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Ceph Sink"
+    description: |-
+      Upload data to an Ceph Bucket managed by a Object Storage Gateway.
+
+      In the header, you can optionally set the `file` / `ce-partition` property to specify the name of the file to upload.
```
Comment (oscerd) [inline library/camel-kamelets/src/main/resources/kamelets/ceph-sink.kamelet.yaml:36]: ce-file and not ce-partition

**Review by oscerd — CHANGES_REQUESTED** [review]
Minor stuff.

**Pull request merged**